src/Components/Navbar/nav.jsx

```diff
@@ -1,28 +1,18 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 
-import route from '../../route';
+import NavItem from './NavItem/navItem';
 
 import style from './nav.module.css';
 
-const NavBar = () => (
-  <nav className={style.nav}>
-    <div className={style.item}>
-      <NavLink to={route.profile} activeClassName={style.active}>Profile</NavLink>
-    </div>
-    <div className={style.item}>
-      <NavLink to={route.dialogs} activeClassName={style.active}>Message</NavLink>
-    </div>
-    <div className={style.item}>
-      <NavLink to={route.news} activeClassName={style.active}>News</NavLink>
-    </div>
-    <div className={style.item}>
-      <NavLink to={route.music} activeClassName={style.active}>Music</NavLink>
-    </div>
-    <div className={style.item}>
-      <NavLink to={route.settings} activeClassName={style.active}>Settings</NavLink>
-    </div>
-  </nav>
-);
+const NavBar = () => {
+  const Items = Object.values(NavItem);
+  const NavItemBlock = Items.map((item) => (<div key={item.id} className={style.item}><NavLink to={item.path} activeClassName={style.active}>{item.name}</NavLink></div>));
```

Comment:
Also, consider adding some indentation to your JSX to improve readability.

Reply:
sure I've added some indentation